### PR TITLE
Add pgroonga_list_broken_indexes() skeleton

### PR DESCRIPTION
**Pull request by abetomo**
Prepare to implement a function that lists the indexes of PGroonga's that may be broken.

### CODE REVIEW

File: data/pgroonga.sql [hunk 1]
```diff
@@ -434,6 +434,12 @@ CREATE FUNCTION pgroonga_wal_set_applied_position()
 	LANGUAGE C
 	STRICT;
 
+CREATE FUNCTION pgroonga_list_broken_indexes()
+	RETURNS void /* todo */
+	AS 'MODULE_PATHNAME', 'pgroonga_list_broken_indexes'
+	LANGUAGE C
+	STRICT
+	PARALLEL SAFE;
```
Comment (kou) [inline data/pgroonga.sql:442]: Could you add the same SQL to `data/pgroonga--3.2.0--3.2.1.sql`?
Could you also add `DROP FUNCTION pgroonga_list_broken_indexes;` to `data/pgroonga--3.2.1--3.2.0.sql`?

Comment (abetomo) [reply data/pgroonga.sql:442]: I was going to add it after it was fixed because there will always be changes, but should I add it already?

Comment (kou) [reply data/pgroonga.sql:442]: Yes, please.
FYI: `data/pgroonga--3.2.1--3.2.0.sql` will not be changed.

Comment (abetomo) [reply data/pgroonga.sql:442]: I got it!

File: src/pgrn-list-broken-indexes.c [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#include "pgroonga.h"
+#include <funcapi.h>
```
Comment (kou) [inline src/pgrn-list-broken-indexes.c:2]:
```suggestion

#include <funcapi.h>
```

**Pull request merged**